admin_guide/manage_nodes.adoc

```diff
@@ -270,3 +270,28 @@ $ oadm manage-node <node1> <node2> \
 Alternatively, instead of specifying specific node names (e.g., `_<node1>_
 _<node2>_`), you can use the `--selector=_<node_selector>_` option to evacuate
 pods on selected nodes.
+
+[[configuring-node-resources]]
+== Configuring Node Resources
+You can configure node resources by adding kubelet arguments to the node
+configuration file:
```

Comment:
Sorry my last comment on this spot got kinda messed up cuz I didn't wrap em in backticks. Basically we need to verify where in this file (e.g., what section/stanza, what parameter name) these should be set.